dist/debian/build_deb.sh

```diff
@@ -81,6 +81,7 @@ pkg_install python3
 pkg_install python3-devel
 pkg_install python3-pip
 
+git config --global --add safe.directory "$BUILDDIR"/scylla-machine-image
```

Comment:
why did you add this?

Reply:
https://github.com/scylladb/scylla-machine-image/runs/6108039418?check_suite_focus=true 

because of that...
Added a Ref in commit description